src/ui/common/src/components/integrations/cards/card.tsx

```diff
@@ -154,6 +178,23 @@ export const IntegrationCard: React.FC<IntegrationProps> = ({
       serviceCard = null;
   }
 
+  let dataStorageInfo, dataStorageText = null;
```

Comment:
I'm going to refactor this into a component and use said component in each of the corresponding cards. This code is very wet :)